### PR TITLE
Edit a returned order - LR - CA - C21, C43A, C45A, C47A

### PR DESCRIPTION
**Pull request by BharadwajSairam**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PRL-4855

### Change description ###

Edit a returned order - LR - CA - C21, C43A, C45A, C47A

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "Name": "Edit a returned order",
+    "Description": "Event to edit order returned by Judge or manager",
+    "DisplayOrder": 50,
+    "PreConditionState(s)": "CASE_ISSUED;JUDICIAL_REVIEW;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/edit-returned-order/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
```
Comment (shashisk97) [inline definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-nonprod.json:15]: judge-or admin?

File: definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-prod.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "Name": "Edit a returned order",
+    "Description": "Event to edit order returned by Judge or manager",
+    "DisplayOrder": 50,
+    "PreConditionState(s)": "CASE_ISSUED;JUDICIAL_REVIEW;PREPARE_FOR_HEARING_CONDUCT_HEARING;DECISION_OUTCOME",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/edit-returned-order/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/judge-or-admin-edit-approve/about-to-submit",
```
Comment (shashisk97) [inline definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-prod.json:15]: judge or admin?

File: definitions/private-law/json/CaseEventToComplexTypes/EditReturnedOrder/SDO/CaseEventToComplexTypes.json [hunk 3]
```diff
@@ -0,0 +1,387 @@
+[
```
Comment (shashisk97) [inline definitions/private-law/json/CaseEventToComplexTypes/EditReturnedOrder/SDO/CaseEventToComplexTypes.json:1]: SDO can not be drafter by solicitor, so do we need these configs, connsidering the size of  PRL ccd config?

File: definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json [hunk 4]
```diff
@@ -62,6 +62,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "18/12/2023",
+    "ID": "OtherOrderDetails",
+    "ListElementCode": "orderCreatedByEmailId",
```
Comment (shashisk97) [inline definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json:68]: Recently this changes was merged to master of storing the creator email id, please check if thats present so you can remove this.

File: definitions/private-law/json/CaseEventToFields/EditReturnedOrder/Page13-SDO/CaseEventToFields.json [hunk 5]
```diff
@@ -0,0 +1,103 @@
+[
```
Comment (shashisk97) [inline definitions/private-law/json/CaseEventToFields/EditReturnedOrder/Page13-SDO/CaseEventToFields.json:1]: ALl SDO changes can be reverted? as we don't need them

File: definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-nonprod.json (hunk 1 above)
Comment (BharadwajSairam) [reply definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-nonprod.json:15]: Its an existing end point which is used for edit and serve and edit and approve events.

File: definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-prod.json (hunk 2 above)
Comment (BharadwajSairam) [reply definitions/private-law/json/CaseEvent/EditReturnedOrder/CaseEvent-prod.json:15]: Its an existing end point which is used for edit and serve and edit and approve events.

File: definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json (hunk 4 above)
Comment (BharadwajSairam) [reply definitions/private-law/json/ComplexTypes/ManageOrders/ComplexTypes.json]: The change in master is for name not for the email

**Pull request merged**